src/compiler/optimize.jl

```diff
@@ -1937,8 +1937,12 @@ function optimize!(mod::LLVM.Module, tm)
         basic_alias_analysis!(pm)
         cfgsimplification!(pm)
         dce!(pm)
+@static if isdefined(LLVM, :cpu_features!)
+        LLVM.cpu_features!(pm)
+else
 @static if isdefined(GPUCompiler, :cpu_features!)
```

Comment:
```suggestion
elseif isdefined(GPUCompiler, :cpu_features!)
```

?

Reply:
does static still apply to the else tho?

that was my worry

Reply:
Yeah it should